WestLaw UK.js

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
That was meant to be a "request changes," not an approval, but there's apparently no way to change it. Other small issue is that we shouldn't have .DS_Store files checked into the repo.

Reply:
All changes _should_ now be made - between writing this originally and now I did the FreeCodeCamp full stack qualification and always felt like I didn't learn much but looking back at what my JS was like before has really made me see the progress I've made 😂